src/lib/PlausibleAnalytics.svelte

```diff
@@ -101,7 +167,13 @@
 
 <svelte:head>
 	{#if enabled}
-		<script data-api={api} data-domain={domain.toString()} defer {src}></script>
+		<script
+			data-api={api}
+			data-domain={domain.toString()}
+			defer
+			src={plausibleSrc}
```

Comment:
#12 identified an issue with [`svelte-preprocess` and the shorthand `{src}`](https://github.com/sveltejs/svelte-preprocess/issues/604)